tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -150,9 +150,46 @@ def start_dhcp_monitor_debug_counter(duthost):
         assert False, "Failed to start dhcpmon in debug counter mode\n"
 
 
+def get_acl_count_by_mark(rand_unselected_dut, mark):
+    output = rand_unselected_dut.shell("iptables -nvL DHCP | grep 'DROP' | grep '{}' | awk '{{print $1}}'"
+                                       .format(mark))
+    pytest_assert(output["rc"] == 0 and len(output["stdout_lines"]) == 1,
+                  "Failed get DHCP acl count for {}, err: {}".format(mark, output["stderr"]))
+    return int(output["stdout"].strip())
+
+
+@pytest.fixture(scope="function")
+def verify_acl_drop_on_standby_tor(rand_unselected_dut, dut_dhcp_relay_data, testing_config, tbinfo):
+    testing_mode, _ = testing_config
+    if testing_mode == DUAL_TOR_MODE and "dualtor-aa" not in tbinfo["topo"]["name"]:
+        pre_client_dhcp_acl_counts = {}
+        for dhcp_relay in dut_dhcp_relay_data:
+            client_interface_name = dhcp_relay["client_iface"]["name"]
+            # Get acl mark per interface
+            output = (rand_unselected_dut
+                      .shell(r"ebtables -L INPUT | grep '\-i {} \-j mark \-\-mark\-set' | awk '{{print $6}}'"
```

Comment:
`iptables`?

Reply:
It's `ebtables`. Mark per interface is added by `ebtables`. Then `iptables` drop packets by mark. In func `get_acl_count_by_mark` get drop count by `iptables`